src/server/qgsserversettings.cpp

```diff
@@ -615,3 +638,23 @@ QString QgsServerSettings::serviceUrl( const QString &service ) const
 
   return result;
 }
+
+int QgsServerSettings::projectCacheCheckInterval() const
+{
+  return value( QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_CHECK_INTERVAL ).toInt();
+}
+
+QString QgsServerSettings::projectCacheStrategy() const
+{
+  QString result = value( QgsServerSettingsEnv::QGIS_SERVER_PROJECT_CACHE_STRATEGY ).toString();
+  if ( !result.compare( QLatin1String( "filesystem" ) ) ||
+       !result.compare( QLatin1String( "periodic" ) ) ||
+       !result.compare( QLatin1String( "off" ) ) )
+  {
+    QgsMessageLog::logMessage( QStringLiteral( "Invalid cache strategy, using 'filesystem' as default." ), "Server", Qgis::MessageLevel::Warning );
```

Comment:
maybe useful to list the allowed values in the message.

Reply:
Done